crypto3/libs/zk/include/nil/crypto3/zk/snark/systems/plonk/placeholder/lookup_argument.hpp

```diff
@@ -745,8 +763,9 @@ namespace nil {
                         std::vector<typename FieldType::value_type> lookup_input;
                         for( std::size_t g_id = 0; g_id < lookup_gates.size(); g_id++ ){
                             const auto &gate = lookup_gates[g_id];
+                        typename FieldType::value_type selector_value;
                             auto key = std::tuple(gate.tag_index, 0, plonk_variable<typename FieldType::value_type>::column_type::selector);
-                            auto selector_value = evaluations[key];
+                            selector_value = evaluations[key];
```

Comment:
If you want to explicitly specify the type, you can do it here:

typename FieldType::value_type selector_value = evaluations[key];